src/main/scala/com/databricks/spark/xml/util/TypeCast.scala

```diff
@@ -22,10 +22,11 @@ import java.util.Locale
 
 import scala.util.Try
 import scala.util.control.Exception._
-
 import org.apache.spark.sql.types._
 import com.databricks.spark.xml.XmlOptions
 
+import scala.util.control.NonFatal
```

Comment:
ditto for import style - https://github.com/databricks/scala-style-guide#imports